### PR TITLE
Relax more rules and add Composer scripts

### PR DESCRIPTION
**Pull request by antograssiot**
I've pointed some more rules that needed to be relaxed since we are using Zend NamingConvention Sniff while running the CS against Cakephp core.

@ADmad I've added Composer scripts has discussed, I hope that this is it what you had in mind more or less.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -28,5 +28,29 @@
...
+            "sed -i '' 's/<!--<exclude-pattern>\\*\\/tests\\/\\*<\\/exclude-pattern>-->/<exclude-pattern>\\*\\/tests\\/\\*<\\/exclude-pattern>/' CakePHP/ruleset.xml"
+        ],
+        "addStandard" : "phpcs --config-set installed_paths $(pwd)",
+        "test": [
+          "@increaseSeverity",
+          "phpunit",
+          "@resetRuleset"
+        ],
+        "cs-check": [
+            "@addStandard",
```
Comment (ADmad) [inline composer.json]: Not sure if updating the installed_paths and modifying the ruleset file is necessary every time one wants to run the checks.

Comment (antograssiot) [inline composer.json]: I'might not sure for the standard size it is already installed on my machine. Redefiningredients it doesn't hurt I think. Regarding the ruleset it was jut that for this report, given how the tests are written I wanted to stick with all the rules or the CS check but I can remove it (will need to be updated in the Travis config)

**Comment by antograssiot** [comment]
Do not merge this just yet, I polish it later

**Comment by antograssiot** [comment]
ok I've updated it I think it is ok now

File: CakePHP/ruleset.xml [hunk 2]
```diff
@@ -32,13 +32,16 @@
  <rule ref ="Zend.NamingConventions.ValidVariableName.ContainsNumbers">
   <severity>0</severity>
  </rule>
- <rule ref ="Zend.NamingConventions.ValidVariableName.NotCamelCaps">
+ <rule ref ="Zend.NamingConventions.ValidVariableName.StringVarContainsNumbers">
```
Comment (ADmad) [inline CakePHP/ruleset.xml]: Extra space after `ref`

File: CakePHP/ruleset.xml [hunk 3]
```diff
@@ -32,13 +32,16 @@
...
   <severity>0</severity>
  </rule>
- <rule ref ="Zend.NamingConventions.ValidVariableName.MemberVarNotCamelCaps">
+ <rule ref ="Zend.NamingConventions.ValidVariableName.MemberVarContainsNumbers">
```
Comment (ADmad) [inline CakePHP/ruleset.xml]: Extra space after `ref`

File: CakePHP/ruleset.xml [hunk 4]
```diff
@@ -32,13 +32,16 @@
...
   <severity>0</severity>
  </rule>
- <rule ref ="Zend.NamingConventions.ValidVariableName.ContainsNumbers">
+ <rule ref ="Zend.NamingConventions.ValidVariableName.NotCamelCaps">
```
Comment (ADmad) [inline CakePHP/ruleset.xml]: Extra space after `ref`

File: CakePHP/ruleset.xml [hunk 5]
```diff
@@ -32,13 +32,16 @@
...
+  <severity>0</severity>
+ </rule>
+ <rule ref ="Zend.NamingConventions.ValidVariableName.MemberVarNotCamelCaps">
```
Comment (ADmad) [inline CakePHP/ruleset.xml]: Extra space after `ref`

Comment (antograssiot) [inline CakePHP/ruleset.xml]: derp I've only fixed one and didn't check for the others.... 😢

**Pull request merged**